documentation/asciidoc/computers/getting-started/network-installation.adoc

```diff
@@ -51,15 +51,15 @@ You will need a keyboard to make use of the network installation feature. While
 
 video::b1SYVpM9lto[youtube]
 
-You should also insert a **blank** SD Card into the Raspberry Pi's SD Card slot.
+You should also insert a **blank** SD Card into Raspberry Pi computers SD Card slot.
 
 NOTE: You should make sure there isn't a bootable SD Card, or USB drive, attached to your Raspberry Pi.
 
-Power on your Raspberry Pi. As always it will first look for an SD Card, and then a USB drive, to find bootable media. However if you have a keyboard attached the Raspberry Pi will now show the network installation screen.
+Power on your Raspberry Pi. As always, it will first look for an SD Card, and then a USB drive, to find bootable media. However, if you have a keyboard attached, Raspberry Pi will now show the network installation screen.
 
 image::images/network-install-1.png[width="80%"]
 
-In the background the Raspberry Pi is still looking for a bootable imagem, but you can now start a network installation by holding down the `SHIFT` key for 3 seconds. Confirm that you want to go ahead by pressing `SPACE`, and you will be prompted to connect your Raspberry Pi to the network with an Ethernet Cable. 
+In the background Raspberry Pi is still looking for a bootable imagem, but you can now start a network installation by holding down the `SHIFT` key for 3 seconds. Confirm that you want to go ahead by pressing `SPACE`, and you will be prompted to connect your Raspberry Pi to the network with an Ethernet Cable. 
```

Comment:
Not good grammar. Can't just remove the "the" in the sentence, no longer parses.